frontend/src/metabase/lib/dom.js

```diff
@@ -349,8 +366,21 @@ const getOrigin = url => {
 const getLocation = url => {
   try {
     const { pathname, search, hash } = new URL(url, window.location.origin);
+    const sitePath = getSitePath().split("/")[1];
```

Comment:
This is error prone and hard to read/understand. What part of the path do you need here?

Reply:
I need the site path without the `/`. Maybe I should just use `getSitePath().slice(1)` instead?

Since I'm only comparing the site path with the pathname that users are visiting. it's possible that `sitePath` from `site-url` settings would be `/Metabase` but users are on `http://localhost:3000/metabase` (lowercase `m`)

Reply:
Tested that `getSitePath().slice(1)` doesn't work when users set metabase site url with subpath ending with a slash e.g. `localhost:3000/metabase/`

getSitePath().split("/") is guaranteed to be a string because `getSitePath` will either return `"/"` or `"/subPathWithOrWithoutSlash"`.